### ISSUE TITLE
Update start workflows one this action's ready for consumption

### ISSUE THREAD

**Issue by paulo-ferraz-oliveira** [erlef/setup-erlang#6]
Here: https://github.com/actions/starter-workflows/blob/b2e786d4e9af21f57ab07c1bc7231ed867f7675f/ci/erlang.yml, to use `erlef`'s action.

Are there any other places we can update this also, so it's somewhat consistent?

Should we probably ask `gleam-lang` if it'd be OK to have a notice on their action stating the existence of this one?

**Comment by paulo-ferraz-oliveira** [comment]
@tsloughter, apart from the `starter-workflows` is there any other place you remember worth updating once we have this action up and running?

**Comment by starbelly** [comment]
@paulo-ferraz-oliveira there are other things we need to do before gleam-lang's action can be deprecated, which Louis already said he would when we have a working alternative. Namely, windows, and some other possible platforms, which setup-elixir also doesn't support at the moment.

**Comment by paulo-ferraz-oliveira** [comment]
Right. I forgot Windows support. Here's a ticket for it: https://github.com/erlef/setup-erlang/issues/7.

In that case, this issue probably doesn't need to stay open. Things will eventually happen naturally, hopefully.

**Comment by paulo-ferraz-oliveira** [comment]
Transferring this to `setup-erlang` as per discussion with @starbelly.
